sdk/src/config.rs

```diff
@@ -372,6 +367,7 @@ impl<RwLock: lock_api::RawRwLock> GadgetConfiguration<RwLock> {
     /// * No sr25519 keypair is found in the keystore.
     /// * The keypair seed is invalid.
     #[doc(alias = "sr25519_signer")]
+    #[cfg(any(feature = "std", feature = "wasm"))]
```

Comment:
Did something change in these keystore methods to make them no longer `no_std` compatible?

Reply:
Yes, but I am restructuring to make it `no_std` compatible again. These fixes will be up soon.